src/jest-sonar.js

```diff
@@ -57,6 +57,18 @@ class JestSonar {
             return currentPath;
         }, '');
     }
+
+    replaceRootDirInPath(rootDir, filePath) {
+        // Inlined from https://github.com/facebook/jest/blob/6e5b1d60a1214e792b5229993b5475445e9c1a6e/packages/jest-config/src/utils.ts#L57-L69
```

Comment:
Please remove comments from the code you can add this to the body of the commit message

Reply:
Amended commit message accordingly 👍